src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -82,6 +82,7 @@ Flags:
 --access_key <string>                                     (optional)                        Set the access key for the account (default is generated)
 --secret_key <string>                                     (optional)                        Set the secret key for the account (default is generated)
 --fs_backend <none | GPFS | CEPH_FS | NFSv4>              (optional)                        Set the filesystem type of new_buckets_path (default config.NSFS_NC_STORAGE_BACKEND)
+--bucket_creation <allow | block>                         (optional)                        Set the account to explicitly allow or block bucket creation
```

Comment:
I will change it.
The open question is about the rest of the boolean flags that we have - if we wish to have the same solution.
If so, it will be in a different PR (after this one will be merged).

Reply:
np thanks!

Reply:
@shirady is not related to this, Can we reduce the space between the `flag`, option and the description? In terminal because of the huge space description is coming in the next line.

Reply:
@naveenpaul1 we can change it of course.
The space between the flag and the (optional) was set due to this line:
https://github.com/noobaa/noobaa-core/blob/563e0ab66a9292da2071e3cc5849f3c861e6ed63/src/manage_nsfs/manage_nsfs_help_utils.js#L69

We can reduce the spaces between the (optional) and the description (in a different PR).

Reply:
@romayalon I didn't understand you - do we want the other boolean option also to be able to send true/false and not just the option? (in a different PR as mentioned).

Reply:
I think it's a bit weird to fail the command when  specifying --show_secrets=true

Reply:
I will change that, thanks.